robolectric/src/main/java/org/robolectric/internal/EnvHolder.java

```diff
@@ -1,22 +1,45 @@
 package org.robolectric.internal;
 
-import org.robolectric.manifest.AndroidManifest;
-import org.robolectric.res.FsFile;
+import org.robolectric.internal.bytecode.InstrumentingClassLoader;
+import org.robolectric.internal.bytecode.InstrumentingClassLoaderConfig;
+import org.robolectric.internal.dependency.DependencyResolver;
+import org.robolectric.util.Pair;
 
-import java.lang.ref.SoftReference;
-import java.util.HashMap;
+import java.net.URL;
+import java.util.LinkedHashMap;
 import java.util.Map;
 
 public class EnvHolder {
```

Comment:
I'll probably rename this class in a follow up PR, perhaps InstrumentingClassloaderFactory ?